src/hooks/useDSHeading.tsx

```diff
@@ -36,9 +38,26 @@ function useDSHeading({
       // passed, clone the element and add the styles to the cloned element.
       // This is to account for base default styles for a component that may
       // be ignored in a custom heading.
-      updatedTitle = additionalStyles
-        ? React.cloneElement(title, { __css: additionalStyles })
-        : title;
+      if (title.type === Heading) {
+        updatedTitle = additionalStyles
+          ? React.cloneElement(title, { __css: additionalStyles })
+          : title;
+      } else if (headingList.includes(title.type)) {
+        // If the user passed in an HTML heading element, just use that.
+        // They will not get any DS styling.
+        updatedTitle = title;
+        console.warn(
+          "NYPL Reservoir useDSHeading: An HTML heading element was passed " +
+            "for the `title` or `headingText` in a component. This will render " +
+            "without DS-specific styling."
+        );
+      } else {
+        console.warn(
+          "NYPL Reservoir useDSHeading: A DS `Heading` component or an HTML " +
```

Comment:
Similarly...
```
NYPL Reservoir useDSHeading: An unsupported heading element was passed to the `title` or `headingText` prop in a Reservoir component, so that component's title will not be rendered. Instead, a DS `Heading` component or an HTML heading element should be passed. 
```